gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
 {% import "%namespace/%name_%version/%sub/services/%service/_client_macros.j2" as macros %}
+{% import "%namespace/%name_%version/%sub/services/%service/_shared_macros.j2" as shared_macros %}
```

Comment:
`{% import ... %}` is not transitive? `_shared_macros` is imported by `_client_macros`, but we have to explicitly import the former here?

Reply:
Correct, I get an error if I don't include it.

```
jinja2.exceptions.UndefinedError: 'shared_macros' is undefined
```

Reply:
Bummer. Oh, well.